server/controllers/userAuth.js

```diff
@@ -237,15 +231,7 @@ const resetPassword = (req, resp, next) => {
           next();
         });
     })
-    .catch(err => {
-      const { message } = err;
-
-      if (message) {
-        return resp.status(400).send({ message });
-      }
-
-      resp.sendStatus(400);
-    });
+    .catch(() => resp.sendStatus(400));
```

Comment:
Shouldn't we always send status OK? (E.g if there is no user also)

Reply:
if there is no user we send OK status